db/seeds.d/010-calibrate_bcrypt_cost.rb

```diff
@@ -0,0 +1,11 @@
+# calibrate default cost to be at least 5ms (performance-wise safe)
```

Comment:
This whole block should be behind an `if SETTINGS[...]` so admins can always override this. This also makes it easy to use a cost of 1 in development mode for example.

Reply:
In test environment, cost is set to 1 automatically. I kinda like to have development environment as close as production if it makes sense and here I think that it does. If there was regression in calibration code, we would immediately see this if that code is same. Not sure if 1 or 5 ms makes things any better during development.